test/basic.spec.js

```diff
@@ -138,6 +142,17 @@ describe("Basic Suite", function() {
             expect(testComponents.length).toEqual(1);
         });
 
+        it('can see inside the shadowRoot with ">" in selector', function() {
+            const testComponent = createTestComponent(parent, {
+                childClassName: 'header-1',
+                internalHTML: '<div class="header-2"><div class="find-me"></div></div>'
+            });
+            testComponent.shadowRoot.querySelector('.header-2').host = "test.com";
+            testComponent.classList.add('container');
+            const testComponents = querySelectorAllDeep(`.container > div > .header-2 > .find-me`);
+            expect(testComponents.length).toEqual(1);
```

Comment:
worth confirming that testsComponents[1] is the element we expect e.g. check its classList has find-me